### ISSUE TITLE
Linux Build

### ISSUE THREAD

**Issue by liberodark** [vinaychandranvs/subgenesis#2]
Hi

Please make a linux build in deb x64.

Cordialy

exemple : https://github.com/liberodark/ODrive/blob/master/package.json

**Comment by vinaychandranvs** [comment]
Yep. I don't have a Linux machine to test. If you can help with testing, I can create a Binary and send it to you. @liberodark

**Comment by liberodark** [comment]
Yes is ok for me send me here im try it !
